pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -255,22 +255,33 @@ message SplitAndScatterSpec {
   repeated TableRekey rekeys = 2 [(gogoproto.nullable) = false];
 }
 
-// FileCompression list of the compression codecs which are currently
-// supported for CSVWriter spec
+// FileCompression is a list of the compression codecs which are currently
+// supported for Exporter spec.
 enum FileCompression {
   None = 0;
   Gzip = 1;
+  Snappy = 2; // Only available for Parquet.
 }
 
-// CSVWriterSpec is the specification for a processor that consumes rows and
-// writes them to CSV files at uri. It outputs a row per file written with
+// FileFormat is list of file formats which are currently
+// supported for the Exporter spec.
+/*enum FileFormat {
+  Csv = 0;
+  Parquet = 1;
+}*/
+
+// ExporterSpec is the specification for a processor that consumes rows and
+// writes them to Parquet or CSV files at uri. It outputs a row per file written with
 // the file name, row count and byte size.
 message CSVWriterSpec {
```

Comment:
Ok to modify the name of this message to `ExportSpec`?

Reply:
I believe that at the gRPC and protofbuf level's renaming a message or field is fine (provided we don't make use of it in protobuf.Any) as long as the type remains the same (or is only different in some compatible way)  The main concern is whether other parts of the system depend on the name. For instance, a rename could break an existing query that makes use of `pb_to_json`.  I've started a discussion in slack about this.

Reply:
Done